app/app.go

```diff
@@ -710,21 +767,29 @@ func New(
 	app.EvidenceKeeper = *evidenceKeeper
 
 	govConfig := govtypes.DefaultConfig()
-	app.GovKeeper = govkeeper.NewKeeper(
-		appCodec, keys[govtypes.StoreKey], app.GetSubspace(govtypes.ModuleName), app.AccountKeeper, app.BankKeeper,
-		&stakingKeeper, govRouter, app.MsgServiceRouter(), govConfig,
+	govKeeper := *govkeeper.NewKeeper(
+		appCodec,
+		keys[govtypes.StoreKey],
+		app.AccountKeeper,
+		app.BankKeeper,
+		stakingKeeper,
+		app.MsgServiceRouter(),
+		govConfig,
+		authorityAddr,
 	)
+	govKeeper.SetLegacyRouter(govRouter)
+	app.GovKeeper = *govKeeper.SetHooks(govtypes.NewMultiGovHooks())
```

Comment:
why not just make `app.GovKeeper` a pointer?

Reply:
Fixed in https://github.com/dymensionxyz/dymension/pull/970/commits/392a7b7512d2ea0f9f3ff670e2874a1c84db9b7c